exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/BasicFormatMatcher.java

```diff
@@ -72,7 +72,7 @@ public boolean supportDirectoryReads() {
 
   @Override
   public FormatSelection isReadable(DrillFileSystem fs, FileSelection selection) throws IOException {
-    if (isReadable(fs, selection.getFirstPath(fs))) {
+    if (isFileReadable(fs, selection.getFirstPath(fs))) {
```

Comment:
Why did you change the name here?  If I recall correctly, the Parquet format matcher doesn't necessarily need to read a file and can match at the directory level.